.storybook/preview.js

```diff
@@ -1,4 +1,10 @@
 // .storybook/preview.js
+import { setCustomElements } from '@storybook/web-components';
+
+// funcion que mezcle todos los json
+const counter = require('./custom-elements.json');
+
+setCustomElements(counter);
```

Comment:
Do we need to hardcode all our web components here?

Reply:
no, I just forgot to change the name of the variable xD that was the name I used in a previous test